.gitattributes

```diff
@@ -0,0 +1 @@
+gips_init.tgz filter=lfs diff=lfs merge=lfs -text
```

Comment:
@bhbraswell -- Not sure what this is, but I'm thinking it might be one to pull?

Reply:
Oh jeez yeah that's gotta go. Sorry.